src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
to be honest, I'm not sure how I feel about hiding all these "unused variables". I know it's nice for when you have the warnings turned on, but it also makes it a lot harder to understand the code. It is helpful if when looking at the code, a programmer can see what this argument is supposed to do—and if they need to edit the rule to use the namestore, it will be clear how to do so without having to search around.

So, I kind of feel that we should leave things like this in place. Unused variables is not always a bug.